src/database/migration/1637859360945-create_borrow.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  MigrationInterface,
+  QueryRunner,
+  Table,
+  TableForeignKey,
+} from 'typeorm';
+
+export class createBorrow1637859360945 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.createTable(
+      new Table({
+        name: 'borrow',
+        columns: [
+          {
+            name: 'id',
+            type: 'int',
+            isPrimary: true,
+          },
+          {
+            name: 'borrow',
+            type: 'varchar',
+          },
+          {
+            name: 'estimated_devolution',
```

Comment:
change devolution to return in all places

Reply:
done